plugins/convert_README_to_header.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+
+# A simple script to build a C++ header file containing the plugin description
+# from the plugin's README.md
+# The output file then contains one variable 'plugin_description'.
+
+if [ $# != 2 ]; then
+  echo "Usage: convert_README_to_header.sh <path> <dest file>";
+  echo "<path>: path to plugin dir, e.g. plugins/artnet";
+  echo "<dest file>: e.g. include/ola/plugins/artnet_description.h";
+  exit 1;
+fi
+
+path="$1";
+outfile="$2";
+
+if [ ! -d $path ]; then
+  echo "directory '$path' does not exist";
+  exit 1;
+fi
+
+if [ ! -e "$path/README.md" ]; then
+  echo "README.md file in '$path' does not exist";
+  exit 1;
+fi
+
+plugin=`basename "$path"`;
+outfilename=`basename $outfile`;
+
+desc=`sed -e ':a' -e 'N' -e '$!ba' -e 's/\n/\\\\n"\\
```

Comment:
What a daft choice on the Mac front. Can we have a link to the Stackoverflow article in a comment here, to explain the craziness please.